app/api/migrations/migrations/58-add_system_key_translations/index.js

```diff
@@ -32,60 +38,24 @@ async function insertSystemKeys(db, newKeys) {
     });
   });
 
-  await translations.forEach(tr => db.collection('translations').replaceOne({ _id: tr._id }, tr));
+  await Promise.all(
+    translations.map(tr => db.collection('translations').replaceOne({ _id: tr._id }, tr))
+  );
 }
 
 export default {
   delta: 58,
 
   name: 'add_system_key_translations',
 
-  description: 'Inserts system keys of added translations.',
+  description: 'Adding missing translations for system keys, through importing from a csv file.',
 
   async up(db) {
     process.stdout.write(`${this.name}...\r\n`);
-    const systemKeys = [
-      {
-        key: 'Suggestion',
-        value: 'Suggestion',
-      },
-      {
-        key: 'All',
-        value: 'All',
-      },
-      {
-        key: 'Filled',
-        value: 'Filled',
-      },
-      {
-        key: 'Empty',
-        value: 'Empty',
-      },
-      {
-        key: 'Segment',
-        value: 'Segment',
-      },
-      {
-        key: 'State',
-        value: 'State',
-      },
-      {
-        key: 'Reviewing',
-        value: 'Reviewing',
-      },
-      {
-        key: 'Dashboard',
-        value: 'Dashboard',
-      },
-      {
-        key: 'Find suggestions',
-        value: 'Find suggestions',
-      },
-      {
-        key: 'per page',
-        value: 'per page',
-      },
-    ];
-    await insertSystemKeys(db, systemKeys);
+
```

Comment:
What was the issue with the non-file based approach? I sort of like that better, since it avoids importing the csv facilities.

Reply:
I changed it because since it increased the number of translations, the file ended to be so long

Reply:
Make sense; same reason we originally did it.